test/test-rpc_secure.js

```diff
@@ -0,0 +1,47 @@
+var server = require('../src/server');
+var bonescript = require('../src/bonescript');
+var myserver = null;
+
+exports.setUp = function (callback) {
+    server.serverStart(8000, process.cwd(), { // create a secure server by supplying credentials
+        username: 'testuser',
```

Comment:
For "production" use, these need to be stored statically in a file. Where is the associated update to server.js (not src/server.js)? That file should be updated to read all parameters from a configuration file. Perhaps use /etc/default/bonescript?

Reply:
Sure, i will modifications for that, will use /etc/default/bonescript for the default config. I did not modify the server.js till now as i noticed that it starts the server with default settings, will make changes to it soon.